source/classes.tex

```diff
@@ -1509,7 +1505,8 @@
 \end{example}
 
 \pnum
-A non-template constructor for class \tcode{X} is a move constructor if its
+\indextext{move!class object|see{constructor, move}}%
+A non-template constructor for class \tcode{X} is a \defnadj{move}{constructor} if its
 first parameter is of type \tcode{X\&\&}, \tcode{const X\&\&},
 \tcode{volatile X\&\&}, or \tcode{const volatile X\&\&}, and either there are
```

Comment:
While we're at it and cleaning up this section, we could say "*cv* `X&&`".

No particular urgency to do it here though, just an idea.